src/tests/intg/test_memory_cache.py

```diff
@@ -621,8 +641,12 @@ def test_initgroups_case_insensitive_with_mc1(ldap_conn,
                                   primary_gid, expected_gids)
 
 
+@pytest.mark.converted('test_id.py', 'test_id__fq_names_case_insensitive')
+@pytest.mark.converted('test_id.py', 'test_id__case_insensitive')
+@pytest.mark.converted('test_memory_cache.py', 'test_memory_cache__fq_names_case_insensitive')
+@pytest.mark.converted('test_memory_cache.py', 'test_memory_cache__case_insensitive')
 def test_initgroups_case_insensitive_with_mc2(ldap_conn,
-                                              fqname_case_insensitive_rfc2307):
+                                              fqname_case_insensitive_rfc2307fqname_case_insensitive_rfc2307):
```

Comment:
Please revert this change. This test now fails.

Reply:
Oops. I probably missclicked Ctrl+V.